metriks/metriks.go

```diff
@@ -104,7 +104,9 @@ func createDatadogSink(url string, name string, tags map[string]string, extraTag
 	}
 
 	sink.SetTags(ddTags)
-
+	if err := initDistribution(url, name, ddTags); err != nil {
```

Comment:
It doesn't look like this `name` field is the serviceName?

Reply:
wow - yeah this whole init section is a bit of a mess. But that is a separate thing. Let me fix that (that's what I get for making some assumptions!)